src/main/java/com/cloudbees/jenkins/support/BundleBrowser.java

```diff
@@ -0,0 +1,108 @@
+package com.cloudbees.jenkins.support;
```

Comment:
@oleg-nenashev It's probably best we ask the mentor mailing list about this. Personal preference is to keep everything in the same package (searching gets harder if it is split).